bats/tests/containers/platform.bats

```diff
@@ -1,7 +1,6 @@
 setup() {
     load '../helpers/load'
 }
-
```

Comment:
Gratuitous whitespace change; no need to edit this file.

Reply:
done

Reply:
Doesn't look like it. 😄